Documentation/contributing/testing/unit.rst

```diff
@@ -9,9 +9,24 @@
 Integration Testing
 ===================
 
-Cilium uses the standard `go test <https://golang.org/pkg/testing/>`__ framework
-in combination with `gocheck <http://labix.org/gocheck>`__ for richer testing
-functionality.
+Cilium uses the standard `go test <https://golang.org/pkg/testing/>`__ framework.
+All new tests `must use testing`_.
```

Comment:
Edits for clarity, specific is better than general. This also affects link formatting 👇🏻 
```suggestion
All new tests must use `the standard test framework`_.
```